_posts/en/newsletters/2025-05-16-newsletter.md

```diff
@@ -0,0 +1,161 @@
+---
+title: 'Bitcoin Optech Newsletter #354'
+permalink: /en/newsletters/2025/05/16/
+name: 2025-05-16-newsletter
+slug: 2025-05-16-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a fixed vulnerability affecting old
+versions of Bitcoin Core.  Also included are our regular sections
+summarizing recent discussions about changing Bitcoin's consensus rules,
+announcing new releases and release candidates, and describing notable
+changes to popular Bitcoin infrastructure software.
+
+## News
+
+- **Vulnerability disclosure affecting old versions of Bitcoin Core:**
+  Antoine Poinsot [posted][poinsot addrvuln] to the Bitcoin-Dev mailing
+  list to announce a vulnerability affecting Bitcoin Core versions
+  before 29.0.  The vulnerability was originally [responsibly
+  disclosed][topic responsible disclosures] by Eugene Siegel along with
+  another closely related vulnerability described in [Newsletter
+  #314][news314 excess addr].  An attacker could send an excessive
+  number of node address advertisements to force a 32-bit identifier to
+  overflow, resulting in a node crash.  This was partly
+  mitigated by limiting the number of updates to one per peer per every
+  ten seconds, which for the default limit of about 125 peers would
+  prevent overflow unless the node was continuously attacked for over 10
+  years. <!-- 2**32 * 10 / 125 / (60 * 60 * 24 * 365) --> The
+  vulnerability was fixed in Bitcoin Core 29.0, released last month.
```

Comment:
How about specifying that the vulnerability was fixed by making the identifier a 64-bit identifier?